docs/CONTRIBUTING.md

```diff
@@ -111,6 +114,24 @@ The usage of host.docker.internal comes from the [Docker networking documentatio
 
 There is currently no arm64 support for the go1.x runtime, see [here](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html). Known issues running on linux/arm64 include the inability to network with the localhost from the public.ecr.aws/lambda/go Docker image.
 
+### Testing in Docker
+
+We support local testing in Docker. Ensure ``docker`` is installed and running, and then run:
+
+```bash
+bash etc/run_docker.sh
```

Comment:
It could be helpful to make this a make target?

```Makefile
.PHONY: build-tests-docker
build-tests-docker:
	etc/run_docker.sh
```

Reply:
It gets tricky with how to pass args.  I thought we eventually want to remove the Makefile altogether?

Reply:
Gotcha, that makes sense. Thanks!